internal/use_cases/create_pull_request.go

```diff
@@ -43,8 +46,34 @@ type CreatePullRequest struct {
 	BranchProvider          domain.BranchProvider
 }
 
+func (cpr *CreatePullRequest) validateTemplateFile() error {
+	if cpr.Cfg.TemplatePath == "" {
+		return nil // No template to validate
+	}
+
+	templatePath := cpr.Cfg.TemplatePath
+	if !filepath.IsAbs(templatePath) {
+		repoRoot, err := cpr.Git.GetRepositoryRoot()
+		if err != nil {
+			return fmt.Errorf("failed to determine repository root: %w", err)
+		}
+		templatePath = filepath.Join(repoRoot, templatePath)
+	}
+
+	if _, err := os.Stat(templatePath); os.IsNotExist(err) {
+		return fmt.Errorf("template file does not exist: %s", cpr.Cfg.TemplatePath)
+	}
+
+	return nil
```

Comment:
thought: maybe we can extract this duplicated logic into a `normalizeTemplatePath` function or similar that returns the template path as absolute path and check if exists directly and call that function inside validate template.

Something like `func normalizeTemplatePath(path string) (normalizedPath string, err error)`